faunadb-java-dsl/src/main/java/com/faunadb/client/types/Field.java

```diff
@@ -106,6 +106,38 @@ public CollectionCodec(Path path, Field<A> field) {
     return new Field<>(Path.empty(), codec);
   }
 
+  /**
+   * Creates a field that converts its underyling value to the {@link Class} provided.
+   *
+   * @param <T> the desired final type
+   * @return a new {@link Field} instance
+   */
+  public static <T> Field<T> as(Class<T> clazz) {
+    return new Field<>(Path.empty(), new Codec<T>() {
```

Comment:
Isn't this the same as `root().to(clazz)`?

Reply:
I suppose so. But is that the semantics you wanted?

Reply:
Not sure if you got your question. What I meant is that the method signature looks good. Just its implementation could be shorter.

Reply:
I wasn't clear from the ticket what exactly this method should _do_, so I guessed. If I guessed correctly, I will shorten the implementation. If I guess incorrectly, please correct me. :)